Fixed `/export` for timestamp, agent name (#854)
jupyterlab/jupyter-ai: docs/source/users/index.md

```

### Exporting chat history

> Use the `/export` command to export the chat history from the current session to a markdown file named `chat_history-YYYY-MM-DD-HH-mm-ss.md`. Using `/export <file_name>` will export the chat history to `<file_name>` without the timestamp instead. You can export chat history as many times as you like in a single session. Each successive export will include the entire chat history up to that point in the session.

Reviewer: ```suggestion
Use the `/export` command to export the chat history from the current session to a markdown file named `chat_history-YYYY-MM-DD-HH-mm-ss.md`. You can also specify a filename using `/export <file_name>`. Each export will include the entire chat history up to that point in the session.
```
